### ISSUE TITLE
Disable activity for E2E folders

### ISSUE THREAD

**Issue by georgehrke** [nextcloud/end_to_end_encryption#144]
Otherwise your activity stream will show something like:
```
You created 15437a6aec794675ad644848da3f58c0, d5bf4bb2ee794929886d8f844595db38, bff1a6b99bef445baa8ca711d4c52e14, 8b4b544fe6a14cd1b2bcd52889cdd808 and 763d82e1f85c41bfa9ae344cfa1fef56
```

**Comment by georgehrke** [comment]
@nickvergessen Is there any event i can use to suppress activity for certain  files? :)

**Comment by nickvergessen** [comment]
https://github.com/nextcloud/server/blob/master/apps/files/lib/Activity/Provider.php#L401

Make sure this still matches or adjust it

**Comment by georgehrke** [comment]
Putting on hold until #150 is merged, as it will change the filenames.

**Comment by sunjam** [comment]
I'm confused... is there somewhere else the user can confirm their encrypted files are being synced besides activity feed on desktop and mobile apps?

**Comment by joshtrichards** [comment]
This was originally reported in #45 and implemented server side in nextcloud/server#8298. Also RFC was updated in nextcloud/end_to_end_encryption_rfc#15 based on  nextcloud/end_to_end_encryption_rfc#13 to better standardize naming across clients to ease parsing.

So if we're seeing that in the wild something is broken here:

https://github.com/nextcloud/server/blob/1fe951f0026eaa512048d6f60481bbbf080172cc/apps/files/lib/Activity/Provider.php#L430

The example looks like a 32 character compliant string. Maybe this was still on a really old NC version? (This was early 2020 that @georgehrke reported this).

I don't see other reports of this. So maybe this not valid any longer and #45 really did close this out. :)

Has anyone seen this in the wild in the past few years? I'm going to close this out on the assumption of "no". Feel free to reopen if someone sees it again...
